Velocity4LumiContainers/images/xcclplugin/specs.yaml

```diff
@@ -0,0 +1,41 @@
+versions:
+  - spec:
+    - 0.1
+    - 0.2
+    - 0.3
+variables:
+  - name: git_hash
+    value: 17d41cb
+    when: xcclplugin@0.1
+  - name: git_hash
+    value: 6dae1b0
+    when: xcclplugin@0.2
+  - name: git_hash
+    value: 1ea938a
+    when: xcclplugin@0.3
+  - name: real_version
+    value: aws_ofi_rccl_working
+    when: xcclplugin@0.1
+  - name: real_version
+    value: aws_ofi_rccl_latest
+    when: xcclplugin@0.2
+  - name: real_version
+    value: aws_open_ofi_1.14.x-xccl
+    when: xcclplugin@0.3
```

Comment:
We might want to separate this into separate images for aws-ofi-rccl and open-ofi-xccl if we can reasonably define dependencies that will allow `velocity` to correctly pick one or the other. Though they kind of implement the same thing, they are still different software packages.